service/controller/resource/instance/create_scale_workers.go

```diff
@@ -39,6 +39,12 @@ func (r *Resource) scaleUpWorkerVMSSTransition(ctx context.Context, obj interfac
 
 	allReady, err := vmsscheck.InstancesAreRunning(ctx, r.logger, key.ResourceGroupName(cr), key.WorkerVMSSName(cr))
 	if err != nil {
```

Comment:
I think we use [a different code style](https://github.com/giantswarm/fmt/blob/master/go/errors.md#matching-errors)

```
if vmsscheck.IsVMSSUnsafeError(err) {
	return ScaleUpWorkerVMSS, nil
} else if err != nil {
	return "", microerror.Mask(err)
}
```

Reply:
thanks, fixed